src/frontend/api/fagsak.ts

```diff
@@ -80,11 +80,23 @@ export const apiOpprettBehandling = (data: IOpprettBehandlingData) => {
 };
 
 export interface IRestVilkårsvurdering {
-    resultat: BehandlingResultat;
-    samletVilkårResultat: IVilkårResultat[];
+    brevType: BehandlingResultat;
```

Comment:
Får se an hva som skjer med brevType backend, men uansett burde man kanskje endre litt slik at man får BrevType istedet for BehandlingResultat isåfall.